src/crypto.rs

```diff
@@ -31,17 +31,10 @@ pub type KdfParams = scrypt::ScryptParams;
 
 lazy_static! {
     /// [`KdfParams`] suitable for production use.
-    pub static ref KDF_PARAMS_PROD: KdfParams = scrypt::ScryptParams::recommended();
+    static ref KDF_PARAMS_PROD: KdfParams = scrypt::ScryptParams::new(15, 8, 1).unwrap();
```

Comment:
What's the reason to not go for `recommended`? Are these values based on the recommendation or a certain target we want to hit?